docs/guides/lockup/03-gas-benchmarks.md

```diff
@@ -27,7 +27,7 @@ view the code that generates these benchmarks.
 | `renounce`                                                  | 19,381    |
 | `createWithDurations` (Broker fee set) (cliff not set)      | 129,276   |
 | `createWithDurations` (Broker fee not set) (cliff not set)  | 113,680   |
-| `createWithDurations` (Broker fee set) (cliff set)          | 138,071   |
+| `createWithDurations` (Broker fee set) (cliff set)          | 138,71   |
```

Comment:
The formatter seems to have been formatting it incorrectly.

Reply:
Oh, good catch! I've updated the route to gas-benchmarks and the script ran on top of it.

Looking at the code now, I think the regex was only configured for non-padded values, meaning `138,071` was treated as `138` and `071`.

https://github.com/sablier-labs/docs/blob/9eedfa220743e9b21d9855f9fe8b6c73e0436545/formatter.js#L5C52-L5C62